front/app/containers/IdeasNewPage/IdeasNewIdeationForm/index.tsx

```diff
@@ -175,10 +175,8 @@ const IdeasNewIdeationForm = ({ project, phaseId }: Props) => {
       anonymous: postAnonymously ? true : undefined,
     });
 
-    const ideaId = idea.data.id;
     participationMethodConfig?.onFormSubmission({
       project: project.data,
-      ideaId,
```

Comment:
Removed this unnecessary property, you can already derive the idea id from the idea itself